docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
-+        cloud: ec2
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no
+$ teleport-windows-auth-setup.exe install --cert=teleport.cer -r
 ```
 
-</Details>
-
-(!docs/pages/includes/start-teleport.mdx service="the Teleport Desktop Service"!)
-
-## Step 3/4. Configure Windows access
-
-In order to gain access to a remote desktop, a Teleport user needs to have the
+## Step 2/4. Configure the Windows Desktop Service
+
+Now that you have prepared the Windows computer to enroll in the cluster, the next steps
+involve:
+
+- Generating an invitation token on your administrative workstation.
+- Configuring a Linux server to run the Windows Desktop Service.
+
+To configure the Windows Desktop Service:
+
+1. Sign in to the Teleport cluster from your administrative workstation to authenticate your identity.
+
+1. Generate a short-lived invitation token by running the following command:
+   
+   ```code
+   $ tctl tokens add --type=windowsdesktop
+   ```
+   
+   The command displays information similar to the following:
+
+   ```code
+   The invite token: (=presets.tokens.first=)
+   This token will expire in 60 minutes.
+   
+   This token enables Desktop Access.  See https://goteleport.com/docs/desktop-access/
+   for detailed information on configuring Teleport Desktop Access with this token.
+   ```
+
+1. Copy the token from your administrative workstation and paste it into a file on the 
+Linux server where you want to run the Windows Desktop Service.
+   
+   For example, create a file named `/tmp/token` on the Linux server and paste the token
+   into the file.
+
+1. Check whether Teleport is installed on the Linux server where you want to run the 
+Windows Desktop Service by running the following command:
+   
+   ```code
+   $ teleport version
+   ```
+
+   If Teleport isn't installed on the Linux server, follow the appropriate [Installation
+instructions](../installation.mdx#installation-instructions) for your environment.
+
+1. Configure the `/etc/teleport.yaml` file on the Linux server with settings similar to the 
+following for the Windows Desktop Service: 
+   
+   ```yml
+   version: v3
+   teleport:
+     nodename: <Var name="windows.teleport.example.com" />
+     proxy_server: <Var name="teleport.example.com:443" />
+     auth_token: <Var name="/tmp/token" />
+   windows_desktop_service:
+     enabled: yes
+     non_ad_hosts:
+     - 192.0.2.156
+   auth_service:
+     enabled: no
+   proxy_service:
+     enabled: no
+   ssh_service:
+     enabled: no
+   ```
+   
+   In this file:
+   
+   - Set the `proxy_server` to the address of your Teleport cluster.
+   - List the Windows IP addresses or DNS names under `non_ad_hosts`.
+   
+   Teleport can't automatically discover Windows desktops without Active Directory. 
+   Therefore, you must specify the Windows IP addresses or DNS name that you want to 
+   enroll using the Teleport configuration file or use the Teleport [API](../api/introduction.mdx) 
```

Comment:
For how-to guides, the style guide recommends not linking outside the guide if it's not necessary: https://goteleport.com/docs/contributing/documentation/style-guide/#expectations

I think the second link in this paragraph is okay, but I don't think we need this link.